core/src/test/java/hudson/scheduler/CronTabTest.java

```diff
@@ -301,4 +302,17 @@ public int next(int n) {
             // ok
         }
     }
+
+    @Issue("JENKINS-9283")
+    @Test public void testTimezone() throws Exception {
+        CronTabList tabs = CronTabList.create("TZ=Australia/Sydney\nH * * * *\nH * * * *", Hash.from("seed"));
+        List<Integer> times = new ArrayList<Integer>();
+        for (int i = 0; i < 60; i++) {
+            if (tabs.check(new GregorianCalendar(2013, 3, 3, 11, i, 0))) {
+                times.add(i);
+            }
+        }
+        assertEquals("[35, 56]", times.toString());
```

Comment:
I imagine there should also be a test to actually confirm that the timezone is correctly applied?